pkg/virt-controller/watch/migration.go

```diff
@@ -1020,11 +1020,25 @@ func (c *MigrationController) sync(key string, migration *virtv1.VirtualMachineI
 				return nil
 			}
 			if c.clusterConfig.NonRootEnabled() && util.CanBeNonRoot(vmi) == nil {
+				patches := []string{}
+
 				if vmi.Status.RuntimeUser != 107 {
```

Comment:
Using the `util.NonRootUID` const here would be better as well so that we don't have magic numbers spread all over the place.